pkgs/tools/networking/sslsplit/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, openssl, libevent, libpcap, libnet, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "sslsplit";
+  name = "${pname}-${version}";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    "owner" = "droe";
```

Comment:
No need to quote the left hand values here.

Reply:
Thanks for your suggestion. I've pushed a new commit.